client/src/views/organizations/TOSModal.vue

```diff
@@ -84,7 +84,9 @@ function getLinkForLocale(): string {
 
   window.electronAPI.log(
     'info',
-    `No terms of service link was found matching the locale '${currentLocale}', using '${links.values().next().value}'`,
+    `No terms of service link was found matching the locale \
+'${currentLocale}' in ${JSON.stringify(props.tosLinks.entries())}, \
+using '${links.values().next().value}'`,
   );
   // Still not found, worst cast scenario, return the first value
   return links.values().next().value;
```

Comment:
```suggestion
  const first_link = links.values().next().value;
  window.electronAPI.log(
    'info',
    `No terms of service link was found matching the locale \
'${currentLocale}' in ${JSON.stringify(props.tosLinks.entries())}, \
using '${first_link}'`,
  );
  // Still not found, worst cast scenario, return the first value
  return first_link;
```

I recommend creating the var `first_link` to not repeat the logic to retrieve the default link